<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgAbout</class>
 <widget class="QDialog" name="CDlgAbout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>182</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>441</width>
    <height>182</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>441</width>
    <height>182</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>关于</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>381</width>
     <height>141</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'宋体'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCompany">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>30</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>四川前途高科电子技术有限公司</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblVersion">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>67</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>版本：20130218</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTel">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>94</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>电话：(028)85236997</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFax">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>120</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>传真：(028)85236101</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
